pyproject.toml

```diff
@@ -1,6 +1,3 @@
 [build-system]
-# To get a reproducible wheel, wheel must be pinned to the same version as in
-# dls-python3, and setuptools must produce the same dist-info. Cap setuptools
-# to the last version that didn't add License-File to METADATA
-requires = ["setuptools<57", "wheel==0.33.1"]
+requires = ["setuptools", "wheel"]
```

Comment:
Can we put this back please for back compat

Reply:
Pinning wheel seems to make `build` unhappy. https://github.com/dls-controls/scanspec/runs/6226284583?check_suite_focus=true#step:3:107

Reply:
because you also have it in `setup_requires`. I suggest you have:
pyproject.toml: `requires = ["setuptools<57", "wheel==0.33.1"]`
setup.cfg: `setup_requires: setuptools_scm[toml]>=6.2` and the same in the dev dependency

Reply:
OK I think I have now basically gone in a circle. Is [7c58fc3](https://github.com/dls-controls/scanspec/pull/50/commits/7c58fc39cd72cc469c43bb812a8d130a0ca05a06) functionally different to what I had initially in [198f409](https://github.com/dls-controls/scanspec/pull/50/commits/198f4092b1e3830fad8af86dcae45de1a8069f38)?

Reply:
Pretty much the same. There's an interesting POV here: https://github.com/pypa/setuptools/pull/2824#issuecomment-968464092

Reply:
Does `dls-python3 setup.py bdist_wheel` work with what you have now?